src/Lean/Compiler/ExternAttr.lean

```diff
@@ -61,6 +61,7 @@ private def syntaxToExternAttrData (stx : Syntax) : AttrM ExternAttrData := do
 @[extern "lean_add_extern"]
 opaque addExtern (env : Environment) (n : Name) : ExceptT String Id Environment
 
+-- TODO: documentation
```

Comment:
```suggestion
```

This is implicit on every undocumented symbol. Is there any reason to have the comment on this one in particular?